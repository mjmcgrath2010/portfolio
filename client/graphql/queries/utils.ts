/*
 * Example usage:
 *   export const getStaticProps = () => mapQueriesToProps([{ posts: getPosts }]);
 */
type QueryProps = {
  props: Object;
};

export const mapQueriesToProps = async (queries: any) =>
  await (async () => {
    const data = await queries.reduce(
      async (acc: QueryProps, queryObj: Function) => {
        const [prop, fn] = Object.entries(queryObj)[0];
        try {
          const { props } = acc;
          const data = await fn();
          return {
            props: {
              ...props,
              [prop]: data,
            },
          };
        } catch (e) {
          return {
            props: {
              error: "Oops, something went wrong",
            },
          };
        }
      },
      { props: {} }
    );
    return data;
  })();
