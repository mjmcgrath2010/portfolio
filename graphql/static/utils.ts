export const mapQueriesToProps = async (queries: any) =>
  await (async () => {
    const data = await queries.reduce(
      async (acc, queryObj) => {
        const [prop, fn] = Object.entries(queryObj)[0];
        try {
          const { props } = acc;
          const data = await fn();

          return {
            props: {
              ...props,
              [prop]: data,
            },
          };
        } catch (e) {
          return {
            error: e.message,
          };
        }
      },
      { props: {} }
    );
    return data;
  })();
