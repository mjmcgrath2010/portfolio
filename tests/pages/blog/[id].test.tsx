import BlogId from "@pages/blog/[id]";

import { render, screen } from "@test/utils";

describe("<BlogId />", () => {
  it("matches the snapshot", () => {
    const view = render(<BlogId />);

    expect(view).toMatchSnapshot();
  });
});
