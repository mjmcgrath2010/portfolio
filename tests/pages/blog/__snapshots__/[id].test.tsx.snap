// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogId /> matches the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  position: fixed;
  top: 0;
  left: 0;
  height: 70px;
  width: 100%;
  background: #f6f6f6;
  box-shadow: hsl(0deg 0% 27% / 15%) 2px 3px 5px;
  z-index: 1000;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 0 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 16px;
  height: 100%;
}

.c11 {
  color: rgb(76,86,179);
  cursor: pointer;
  font-size: 16px;
  font-family: 'Titillium Web',sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c11:hover {
  color: rgb(122,130,199);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c11:active,
.c11:visited {
  color: rgb(59,67,140);
}

.c8 {
  color: rgb(76,86,179);
  cursor: pointer;
  font-size: 16px;
  font-family: 'Titillium Web',sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c8:hover {
  color: rgb(122,130,199);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c8:active,
.c8:visited {
  color: rgb(59,67,140);
}

.c1 {
  background: transparent;
  box-shadow: none;
}

.c6 {
  cursor: pointer;
  color: rgb(80,73,73);
  width: 200px;
}

.c6:hover {
  color: rgb(207,109,35);
}

.c3 {
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 16px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4 {
  width: 250px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  justify-self: flex-start;
}

.c4 .c5 {
  color: #fbfbfb;
}

.c4 .c5:visited {
  color: rgb(159,165,215);
}

.c4 .c5:hover {
  color: rgb(159,165,215);
}

.c10 {
  color: rgb(80,73,73);
}

.c10:visited {
  color: rgb(59,67,140);
}

.c10:hover {
  color: rgb(207,109,35);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c7 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  gap: 24px;
}

.c7 .c9 {
  color: #fbfbfb;
}

.c7 .c9:visited {
  color: rgb(159,165,215);
}

.c7 .c9:hover {
  color: rgb(228,151,93);
}

.c13 {
  grid-column: span 12;
  -webkit-align-self: stretch;
  -ms-flex-item-align: stretch;
  align-self: stretch;
  justify-self: center;
}

.c12 {
  display: grid;
  gap: 16px;
  grid-template-rows: auto;
  grid-template-columns: repeat(12,1fr);
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  justify-items: start;
  padding: 24px;
}

<body>
    <div>
      <div
        class="c0 c1"
      >
        <div
          class="c2"
        >
          <div
            class="c3"
          >
            <div
              class="c4"
            >
              <svg
                class="c5 c6"
                fill="none"
                href="/"
                role="button"
                viewBox="0 0 616 66"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M28.704 50.744.806 32.096v-5.328L28.704 8.12v6.882L7.096 29.284v.296l21.608 14.282v6.882ZM36.223 54V5.53h8.584l6.29 19.758 2.146 7.548h.296l2.146-7.548 6.29-19.758h8.584V54h-6.808V32.17c0-1.233.05-2.689.148-4.366.099-1.677.197-3.38.296-5.106.148-1.727.271-3.38.37-4.958.148-1.628.271-3.01.37-4.144h-.222L61.68 24.03l-6.142 17.094H50.95L44.881 24.03l-2.96-10.434H41.7c.148 1.135.271 2.516.37 4.144.148 1.579.271 3.231.37 4.958a91.15 91.15 0 0 1 .296 5.106c.099 1.677.148 3.133.148 4.366V54h-6.66Zm56.803 0V23.734H76.82v-5.92h23.532V54h-7.326Zm3.034-43.216c-1.579 0-2.886-.469-3.922-1.406-1.036-.987-1.554-2.27-1.554-3.848 0-1.579.518-2.837 1.554-3.774C93.174.769 94.481.276 96.06.276c1.579 0 2.886.493 3.922 1.48 1.036.937 1.554 2.195 1.554 3.774 0 1.579-.518 2.861-1.554 3.848-1.036.937-2.343 1.406-3.922 1.406ZM116.825 54V1.534h7.326v34.188h.296l17.834-17.908h8.214L136.287 32.17 152.271 54h-8.066l-12.432-17.316-7.622 7.696V54h-7.326Zm56.359.888c-2.762 0-5.328-.42-7.696-1.258-2.368-.839-4.44-2.072-6.216-3.7-1.776-1.628-3.182-3.626-4.218-5.994-.986-2.368-1.48-5.057-1.48-8.066 0-2.96.518-5.6 1.554-7.918 1.036-2.368 2.393-4.366 4.07-5.994a17.631 17.631 0 0 1 5.92-3.7 18.56 18.56 0 0 1 6.956-1.332c2.664 0 5.032.42 7.104 1.258 2.072.839 3.799 2.023 5.18 3.552 1.431 1.48 2.516 3.28 3.256 5.402.74 2.072 1.11 4.366 1.11 6.882 0 .79-.024 1.53-.074 2.22-.049.69-.123 1.283-.222 1.776H160.9c.395 3.75 1.801 6.537 4.218 8.362 2.467 1.825 5.476 2.738 9.028 2.738 2.023 0 3.873-.271 5.55-.814a26.705 26.705 0 0 0 4.958-2.368l2.59 4.662c-1.874 1.233-3.996 2.27-6.364 3.108-2.318.79-4.884 1.184-7.696 1.184Zm-.888-32.19c-1.381 0-2.688.222-3.922.666a9.625 9.625 0 0 0-3.33 1.85c-.986.839-1.825 1.875-2.516 3.108-.69 1.233-1.184 2.664-1.48 4.292h21.09c-.246-3.305-1.258-5.772-3.034-7.4-1.726-1.677-3.996-2.516-6.808-2.516ZM232.179 54V5.53h8.584l6.29 19.758 2.146 7.548h.296l2.146-7.548 6.29-19.758h8.584V54h-6.808V32.17c0-1.233.049-2.689.148-4.366.098-1.677.197-3.38.296-5.106.148-1.727.271-3.38.37-4.958.148-1.628.271-3.01.37-4.144h-.222l-3.034 10.434-6.142 17.094h-4.588l-6.068-17.094-2.96-10.434h-.222c.148 1.135.271 2.516.37 4.144.148 1.579.271 3.231.37 4.958a91.82 91.82 0 0 1 .296 5.106c.098 1.677.148 3.133.148 4.366V54h-6.66Zm59.689.888c-2.812 0-5.452-.42-7.918-1.258-2.418-.839-4.514-2.072-6.29-3.7-1.776-1.628-3.182-3.601-4.218-5.92-.987-2.368-1.48-5.057-1.48-8.066 0-3.01.542-5.698 1.628-8.066 1.085-2.368 2.565-4.366 4.44-5.994 1.874-1.628 4.02-2.861 6.438-3.7 2.466-.839 5.081-1.258 7.844-1.258 2.96 0 5.55.518 7.77 1.554 2.22.987 4.07 2.195 5.55 3.626l-3.552 4.662c-1.431-1.184-2.911-2.097-4.44-2.738-1.53-.69-3.182-1.036-4.958-1.036-1.924 0-3.7.32-5.328.962-1.628.592-3.034 1.455-4.218 2.59-1.135 1.135-2.023 2.516-2.664 4.144-.642 1.579-.962 3.33-.962 5.254 0 1.924.296 3.7.888 5.328.641 1.579 1.529 2.935 2.664 4.07a12.986 12.986 0 0 0 4.144 2.664c1.578.592 3.33.888 5.254.888 2.22 0 4.193-.42 5.92-1.258a22.727 22.727 0 0 0 4.81-3.034l3.182 4.736a24.25 24.25 0 0 1-6.808 4.144 21.488 21.488 0 0 1-7.696 1.406Zm38.747 0c-3.059 0-5.896-.543-8.51-1.628a19.655 19.655 0 0 1-6.808-4.884c-1.875-2.17-3.355-4.785-4.44-7.844-1.036-3.108-1.554-6.66-1.554-10.656 0-3.947.542-7.474 1.628-10.582 1.085-3.157 2.59-5.821 4.514-7.992a19.075 19.075 0 0 1 6.882-4.958c2.664-1.135 5.55-1.702 8.658-1.702 3.206 0 5.92.641 8.14 1.924 2.22 1.233 4.045 2.59 5.476 4.07l-4.144 4.662a14.681 14.681 0 0 0-3.922-3.034c-1.48-.79-3.33-1.184-5.55-1.184-2.072 0-3.972.444-5.698 1.332-1.727.839-3.207 2.072-4.44 3.7-1.234 1.579-2.196 3.527-2.886 5.846-.691 2.27-1.036 4.835-1.036 7.696 0 5.821 1.208 10.41 3.626 13.764 2.417 3.355 5.92 5.032 10.508 5.032 1.48 0 2.861-.222 4.144-.666 1.332-.444 2.417-1.06 3.256-1.85V34.39h-8.954v-6.068h15.762V49.19c-1.579 1.579-3.651 2.935-6.216 4.07-2.516 1.085-5.328 1.628-8.436 1.628ZM354.932 54V17.814H361l.592 8.436h.222c1.875-2.96 4.144-5.254 6.808-6.882 2.713-1.628 5.698-2.442 8.954-2.442 1.431 0 2.689.123 3.774.37 1.085.197 2.195.567 3.33 1.11l-1.628 6.29a37.083 37.083 0 0 0-3.182-.888c-.888-.197-2.023-.296-3.404-.296-2.615 0-5.131.74-7.548 2.22-2.417 1.48-4.637 4.045-6.66 7.696V54h-7.326Zm46.813.888c-1.727 0-3.355-.247-4.884-.74-1.48-.493-2.763-1.184-3.848-2.072a10.651 10.651 0 0 1-2.59-3.33c-.592-1.283-.888-2.713-.888-4.292 0-2.023.469-3.799 1.406-5.328.987-1.53 2.491-2.837 4.514-3.922 2.072-1.085 4.711-1.973 7.918-2.664 3.256-.69 7.153-1.209 11.692-1.554-.099-2.22-.839-4.12-2.22-5.698-1.332-1.579-3.601-2.368-6.808-2.368-2.22 0-4.391.42-6.512 1.258a44.796 44.796 0 0 0-5.772 2.812l-2.738-4.958a40.957 40.957 0 0 1 3.33-1.776 31.327 31.327 0 0 1 3.922-1.702 33.6 33.6 0 0 1 4.366-1.184 24.472 24.472 0 0 1 4.662-.444c5.032 0 8.806 1.357 11.322 4.07s3.774 6.463 3.774 11.248V54h-5.994l-.592-4.662h-.222c-1.924 1.48-4.095 2.787-6.512 3.922-2.368 1.085-4.81 1.628-7.326 1.628Zm2.072-5.846c1.924 0 3.799-.42 5.624-1.258 1.875-.839 3.749-2.023 5.624-3.552v-8.584c-3.552.296-6.512.69-8.88 1.184-2.368.444-4.267 1.036-5.698 1.776-1.381.69-2.368 1.48-2.96 2.368a5.22 5.22 0 0 0-.888 2.96c0 1.825.691 3.133 2.072 3.922 1.431.79 3.133 1.184 5.106 1.184Zm48.293 5.846c-2.664 0-4.933-.37-6.808-1.11-1.825-.74-3.305-1.776-4.44-3.108-1.085-1.332-1.875-2.91-2.368-4.736-.493-1.875-.74-3.947-.74-6.216V23.734h-9.916v-5.55l10.212-.37.962-11.47h6.068v11.47h17.094v5.92H445.08v15.984c0 3.108.617 5.427 1.85 6.956 1.283 1.53 3.577 2.294 6.882 2.294 1.579 0 3.034-.123 4.366-.37a27.592 27.592 0 0 0 3.922-1.184l1.48 5.402a49.404 49.404 0 0 1-5.18 1.406c-1.825.444-3.922.666-6.29.666ZM468.805 54V1.534h7.326v13.912l-.444 8.14h.222a27.2 27.2 0 0 1 5.698-4.736c2.121-1.283 4.588-1.924 7.4-1.924 4.243 0 7.351 1.258 9.324 3.774 1.973 2.516 2.96 6.142 2.96 10.878V54h-7.326V32.54c0-3.157-.567-5.476-1.702-6.956-1.135-1.53-3.083-2.294-5.846-2.294-1.924 0-3.651.493-5.18 1.48-1.529.937-3.231 2.368-5.106 4.292V54h-7.326Zm78.752 11.84 24.272-64.38h6.364l-24.272 64.38h-6.364Zm40.005-15.096v-6.882L609.17 29.58v-.296l-21.608-14.282V8.12l27.898 18.648v5.328l-27.898 18.648Z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="c7"
            >
              <a
                class="c8 c9 c10"
                href="/about"
              >
                About
              </a>
              <a
                class="c8 c9 c10"
                href="/projects"
              >
                Projects
              </a>
              <a
                class="c8 c9 c10"
                href="/blog"
              >
                Blog
              </a>
              <a
                class="c11 c9 c10"
                href="/components/index.html"
                target="_blank"
              >
                Components
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="c12"
      >
        <div
          class="c13"
        >
          <h1>
            BlogId Page
          </h1>
        </div>
      </div>
    </div>
  </body>,
  "container": .c0 {
  position: fixed;
  top: 0;
  left: 0;
  height: 70px;
  width: 100%;
  background: #f6f6f6;
  box-shadow: hsl(0deg 0% 27% / 15%) 2px 3px 5px;
  z-index: 1000;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 0 24px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 16px;
  height: 100%;
}

.c11 {
  color: rgb(76,86,179);
  cursor: pointer;
  font-size: 16px;
  font-family: 'Titillium Web',sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c11:hover {
  color: rgb(122,130,199);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c11:active,
.c11:visited {
  color: rgb(59,67,140);
}

.c8 {
  color: rgb(76,86,179);
  cursor: pointer;
  font-size: 16px;
  font-family: 'Titillium Web',sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c8:hover {
  color: rgb(122,130,199);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c8:active,
.c8:visited {
  color: rgb(59,67,140);
}

.c1 {
  background: transparent;
  box-shadow: none;
}

.c6 {
  cursor: pointer;
  color: rgb(80,73,73);
  width: 200px;
}

.c6:hover {
  color: rgb(207,109,35);
}

.c3 {
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 16px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4 {
  width: 250px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  justify-self: flex-start;
}

.c4 .c5 {
  color: #fbfbfb;
}

.c4 .c5:visited {
  color: rgb(159,165,215);
}

.c4 .c5:hover {
  color: rgb(159,165,215);
}

.c10 {
  color: rgb(80,73,73);
}

.c10:visited {
  color: rgb(59,67,140);
}

.c10:hover {
  color: rgb(207,109,35);
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c7 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  gap: 24px;
}

.c7 .c9 {
  color: #fbfbfb;
}

.c7 .c9:visited {
  color: rgb(159,165,215);
}

.c7 .c9:hover {
  color: rgb(228,151,93);
}

.c13 {
  grid-column: span 12;
  -webkit-align-self: stretch;
  -ms-flex-item-align: stretch;
  align-self: stretch;
  justify-self: center;
}

.c12 {
  display: grid;
  gap: 16px;
  grid-template-rows: auto;
  grid-template-columns: repeat(12,1fr);
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  justify-items: start;
  padding: 24px;
}

<div>
    <div
      class="c0 c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <div
            class="c4"
          >
            <svg
              class="c5 c6"
              fill="none"
              href="/"
              role="button"
              viewBox="0 0 616 66"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M28.704 50.744.806 32.096v-5.328L28.704 8.12v6.882L7.096 29.284v.296l21.608 14.282v6.882ZM36.223 54V5.53h8.584l6.29 19.758 2.146 7.548h.296l2.146-7.548 6.29-19.758h8.584V54h-6.808V32.17c0-1.233.05-2.689.148-4.366.099-1.677.197-3.38.296-5.106.148-1.727.271-3.38.37-4.958.148-1.628.271-3.01.37-4.144h-.222L61.68 24.03l-6.142 17.094H50.95L44.881 24.03l-2.96-10.434H41.7c.148 1.135.271 2.516.37 4.144.148 1.579.271 3.231.37 4.958a91.15 91.15 0 0 1 .296 5.106c.099 1.677.148 3.133.148 4.366V54h-6.66Zm56.803 0V23.734H76.82v-5.92h23.532V54h-7.326Zm3.034-43.216c-1.579 0-2.886-.469-3.922-1.406-1.036-.987-1.554-2.27-1.554-3.848 0-1.579.518-2.837 1.554-3.774C93.174.769 94.481.276 96.06.276c1.579 0 2.886.493 3.922 1.48 1.036.937 1.554 2.195 1.554 3.774 0 1.579-.518 2.861-1.554 3.848-1.036.937-2.343 1.406-3.922 1.406ZM116.825 54V1.534h7.326v34.188h.296l17.834-17.908h8.214L136.287 32.17 152.271 54h-8.066l-12.432-17.316-7.622 7.696V54h-7.326Zm56.359.888c-2.762 0-5.328-.42-7.696-1.258-2.368-.839-4.44-2.072-6.216-3.7-1.776-1.628-3.182-3.626-4.218-5.994-.986-2.368-1.48-5.057-1.48-8.066 0-2.96.518-5.6 1.554-7.918 1.036-2.368 2.393-4.366 4.07-5.994a17.631 17.631 0 0 1 5.92-3.7 18.56 18.56 0 0 1 6.956-1.332c2.664 0 5.032.42 7.104 1.258 2.072.839 3.799 2.023 5.18 3.552 1.431 1.48 2.516 3.28 3.256 5.402.74 2.072 1.11 4.366 1.11 6.882 0 .79-.024 1.53-.074 2.22-.049.69-.123 1.283-.222 1.776H160.9c.395 3.75 1.801 6.537 4.218 8.362 2.467 1.825 5.476 2.738 9.028 2.738 2.023 0 3.873-.271 5.55-.814a26.705 26.705 0 0 0 4.958-2.368l2.59 4.662c-1.874 1.233-3.996 2.27-6.364 3.108-2.318.79-4.884 1.184-7.696 1.184Zm-.888-32.19c-1.381 0-2.688.222-3.922.666a9.625 9.625 0 0 0-3.33 1.85c-.986.839-1.825 1.875-2.516 3.108-.69 1.233-1.184 2.664-1.48 4.292h21.09c-.246-3.305-1.258-5.772-3.034-7.4-1.726-1.677-3.996-2.516-6.808-2.516ZM232.179 54V5.53h8.584l6.29 19.758 2.146 7.548h.296l2.146-7.548 6.29-19.758h8.584V54h-6.808V32.17c0-1.233.049-2.689.148-4.366.098-1.677.197-3.38.296-5.106.148-1.727.271-3.38.37-4.958.148-1.628.271-3.01.37-4.144h-.222l-3.034 10.434-6.142 17.094h-4.588l-6.068-17.094-2.96-10.434h-.222c.148 1.135.271 2.516.37 4.144.148 1.579.271 3.231.37 4.958a91.82 91.82 0 0 1 .296 5.106c.098 1.677.148 3.133.148 4.366V54h-6.66Zm59.689.888c-2.812 0-5.452-.42-7.918-1.258-2.418-.839-4.514-2.072-6.29-3.7-1.776-1.628-3.182-3.601-4.218-5.92-.987-2.368-1.48-5.057-1.48-8.066 0-3.01.542-5.698 1.628-8.066 1.085-2.368 2.565-4.366 4.44-5.994 1.874-1.628 4.02-2.861 6.438-3.7 2.466-.839 5.081-1.258 7.844-1.258 2.96 0 5.55.518 7.77 1.554 2.22.987 4.07 2.195 5.55 3.626l-3.552 4.662c-1.431-1.184-2.911-2.097-4.44-2.738-1.53-.69-3.182-1.036-4.958-1.036-1.924 0-3.7.32-5.328.962-1.628.592-3.034 1.455-4.218 2.59-1.135 1.135-2.023 2.516-2.664 4.144-.642 1.579-.962 3.33-.962 5.254 0 1.924.296 3.7.888 5.328.641 1.579 1.529 2.935 2.664 4.07a12.986 12.986 0 0 0 4.144 2.664c1.578.592 3.33.888 5.254.888 2.22 0 4.193-.42 5.92-1.258a22.727 22.727 0 0 0 4.81-3.034l3.182 4.736a24.25 24.25 0 0 1-6.808 4.144 21.488 21.488 0 0 1-7.696 1.406Zm38.747 0c-3.059 0-5.896-.543-8.51-1.628a19.655 19.655 0 0 1-6.808-4.884c-1.875-2.17-3.355-4.785-4.44-7.844-1.036-3.108-1.554-6.66-1.554-10.656 0-3.947.542-7.474 1.628-10.582 1.085-3.157 2.59-5.821 4.514-7.992a19.075 19.075 0 0 1 6.882-4.958c2.664-1.135 5.55-1.702 8.658-1.702 3.206 0 5.92.641 8.14 1.924 2.22 1.233 4.045 2.59 5.476 4.07l-4.144 4.662a14.681 14.681 0 0 0-3.922-3.034c-1.48-.79-3.33-1.184-5.55-1.184-2.072 0-3.972.444-5.698 1.332-1.727.839-3.207 2.072-4.44 3.7-1.234 1.579-2.196 3.527-2.886 5.846-.691 2.27-1.036 4.835-1.036 7.696 0 5.821 1.208 10.41 3.626 13.764 2.417 3.355 5.92 5.032 10.508 5.032 1.48 0 2.861-.222 4.144-.666 1.332-.444 2.417-1.06 3.256-1.85V34.39h-8.954v-6.068h15.762V49.19c-1.579 1.579-3.651 2.935-6.216 4.07-2.516 1.085-5.328 1.628-8.436 1.628ZM354.932 54V17.814H361l.592 8.436h.222c1.875-2.96 4.144-5.254 6.808-6.882 2.713-1.628 5.698-2.442 8.954-2.442 1.431 0 2.689.123 3.774.37 1.085.197 2.195.567 3.33 1.11l-1.628 6.29a37.083 37.083 0 0 0-3.182-.888c-.888-.197-2.023-.296-3.404-.296-2.615 0-5.131.74-7.548 2.22-2.417 1.48-4.637 4.045-6.66 7.696V54h-7.326Zm46.813.888c-1.727 0-3.355-.247-4.884-.74-1.48-.493-2.763-1.184-3.848-2.072a10.651 10.651 0 0 1-2.59-3.33c-.592-1.283-.888-2.713-.888-4.292 0-2.023.469-3.799 1.406-5.328.987-1.53 2.491-2.837 4.514-3.922 2.072-1.085 4.711-1.973 7.918-2.664 3.256-.69 7.153-1.209 11.692-1.554-.099-2.22-.839-4.12-2.22-5.698-1.332-1.579-3.601-2.368-6.808-2.368-2.22 0-4.391.42-6.512 1.258a44.796 44.796 0 0 0-5.772 2.812l-2.738-4.958a40.957 40.957 0 0 1 3.33-1.776 31.327 31.327 0 0 1 3.922-1.702 33.6 33.6 0 0 1 4.366-1.184 24.472 24.472 0 0 1 4.662-.444c5.032 0 8.806 1.357 11.322 4.07s3.774 6.463 3.774 11.248V54h-5.994l-.592-4.662h-.222c-1.924 1.48-4.095 2.787-6.512 3.922-2.368 1.085-4.81 1.628-7.326 1.628Zm2.072-5.846c1.924 0 3.799-.42 5.624-1.258 1.875-.839 3.749-2.023 5.624-3.552v-8.584c-3.552.296-6.512.69-8.88 1.184-2.368.444-4.267 1.036-5.698 1.776-1.381.69-2.368 1.48-2.96 2.368a5.22 5.22 0 0 0-.888 2.96c0 1.825.691 3.133 2.072 3.922 1.431.79 3.133 1.184 5.106 1.184Zm48.293 5.846c-2.664 0-4.933-.37-6.808-1.11-1.825-.74-3.305-1.776-4.44-3.108-1.085-1.332-1.875-2.91-2.368-4.736-.493-1.875-.74-3.947-.74-6.216V23.734h-9.916v-5.55l10.212-.37.962-11.47h6.068v11.47h17.094v5.92H445.08v15.984c0 3.108.617 5.427 1.85 6.956 1.283 1.53 3.577 2.294 6.882 2.294 1.579 0 3.034-.123 4.366-.37a27.592 27.592 0 0 0 3.922-1.184l1.48 5.402a49.404 49.404 0 0 1-5.18 1.406c-1.825.444-3.922.666-6.29.666ZM468.805 54V1.534h7.326v13.912l-.444 8.14h.222a27.2 27.2 0 0 1 5.698-4.736c2.121-1.283 4.588-1.924 7.4-1.924 4.243 0 7.351 1.258 9.324 3.774 1.973 2.516 2.96 6.142 2.96 10.878V54h-7.326V32.54c0-3.157-.567-5.476-1.702-6.956-1.135-1.53-3.083-2.294-5.846-2.294-1.924 0-3.651.493-5.18 1.48-1.529.937-3.231 2.368-5.106 4.292V54h-7.326Zm78.752 11.84 24.272-64.38h6.364l-24.272 64.38h-6.364Zm40.005-15.096v-6.882L609.17 29.58v-.296l-21.608-14.282V8.12l27.898 18.648v5.328l-27.898 18.648Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <div
            class="c7"
          >
            <a
              class="c8 c9 c10"
              href="/about"
            >
              About
            </a>
            <a
              class="c8 c9 c10"
              href="/projects"
            >
              Projects
            </a>
            <a
              class="c8 c9 c10"
              href="/blog"
            >
              Blog
            </a>
            <a
              class="c11 c9 c10"
              href="/components/index.html"
              target="_blank"
            >
              Components
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
    >
      <div
        class="c13"
      >
        <h1>
          BlogId Page
        </h1>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
